src/components/editor/SinopiaResourceTemplates.jsx

```diff
@@ -5,6 +5,7 @@ import PropTypes from 'prop-types'
 import { Link } from 'react-router-dom'
 import {BootstrapTable, TableHeaderColumn} from 'react-bootstrap-table'
 import Config from '../../../src/Config'
+import { getGroups, listReourcesInGroupContainer, getResourceTemplate } from '../../sinopiaServer'
```

Comment:
`listReourcesInGroupContainer` import will need to change with correct function name.